scripts/bootstrap.sh

```diff
@@ -26,36 +26,43 @@ apply_firmly(){
 }
 
 install_gitops(){
-  echo ""
-  echo "Installing GitOps Operator."
+  echo
+  echo "Checking if GitOps Operator is already installed"
```

Comment:
Just a tiny suggestion - add the words "and running" since the user may have a cluster where this is installed, but is stopped for some reason or other.

`echo "Checking if the OpenShift GitOps Operator is already installed and running"`